wicket-core/src/main/java/org/apache/wicket/coep/CrossOriginEmbedderPolicyRequestCycleListener.java

```diff
@@ -36,15 +36,15 @@
  * <code>Cross-Origin-Embedder-Policy</code>. The header is not set for the paths that are exempted
  * from COEP. The only valid value of COEP is <code>require-corp</code>, so if the listener is
  * enabled the policy value will be specified as so.
- *
+ * <p>
```

Comment:
I believe, this one will produce JavaDoc warning :(

Reply:
:-( I had a warining locally... And this fixed it. DO we have some IJ settings I could use as default for wicket?

Reply:
> you are right :) no specific warning at this line, sorry for the noise :(

No problem. Everything that makes wicket code better is welcomed :-)

> 
> I'm not using IJ, I believe checkstyle and javadoc maven plugins can be configured to be more strict

Oh! I didn't check the result of those. Just that code build with maven clean install on terminal in my linux box.

At work we have the same "warnings war" because some people use IJ and other use eclipse and we use gradle instead of  maven and it is very diffcult to get all the parties agree about what a warning is :-(